docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '3.0'
```

Comment:
I don't believe this change is required for the named volumes feature; can you explain why you made it?  It breaks using this with the default version of Docker in Ubuntu Xenial

Reply:
No real reason besides moving forward, I can revert this if you like.

Reply:
Yeah, let's revert this; I think the purpose of this version tag is to indicate what features we're requiring, and this doesn't have us use any features that are new in 3.0, so we should keep it at 2.0 for compatibility with as many installations as will work with it.

Reply:
Reverted to 2